src/utils/input_validation/validators.py

```diff
@@ -0,0 +1,177 @@
+import os, sys, uuid 
+
+def validate_room_number(person_id: str) -> bool: 
+    """
+    Check if the user input for room entity is valid
+    """
+    if person_id.isnumeric():
+        return 0 <= int(person_id) <= 9999
+    return False
+
+
+def validate_guest_id(person_id: str) -> bool:
+    """
+    Check if the user input for guest entity is valid.
+    Return proper exit code for special user id's
+    """
+
+    if person_id.isnumeric():
+        return "0" <= person_id <= "9999"
+    return False
+
+
+def validate_credit_card_number(credit_card_number: float) -> bool:
+    """
+    Check if the user input for credit card
+        follows the international card standards.
+    """
+    if credit_card_number.isnumeric():
+        return 8 <= len(credit_card_number) <= 19
+    return False
+
+
+def is_phone_number_valid(phone_number: str):
```

Comment:
Function name is different compared to other validation functions, thus, I'd suggest changing it to validate_phone_number.

As a side note, I'd also suggest annotating the return type for this function for clarity.